src/executions.ts

```diff
@@ -59,6 +59,14 @@ export const runTestCase = (
             );
             break;
         }
+        case 'javascript': {
+            process = spawn(
+                language.compiler,
+                [binPath, ...language.args],
+                // spawnOpts, // `spawnOpts.env` will make PATH be empty, then can't find `node` command
```

Comment:
When testing locally, this line should be commented. So I keep it commented. Don't know why other compilers can be found.

Reply:
Another question is that, `killer` in line 42 will be called very fast (~30ms with default timeout 3000ms) when debugging the extension. Hope it would be called in the correct time in the released version.

Reply:
There is no separate logic for relese/development version. So if the bug exists in dev mode, it would appear in the release mode as well.